src/proto/active_packetizer.rs

```diff
@@ -440,15 +439,14 @@ where
         }
     }
 
+    #[instrument(skip(self, cx, default_watcher))]
```

Comment:
nitpick: To keep the name the same, we should use:

```suggestion
    #[instrument(name = "poll_read", skip(self, cx, default_watcher))]
```